### PR TITLE
DYN-6270/feature/refactor based on sonarcloud reports

### PR DESCRIPTION
**Pull request by Enzo707**
This PR includes refactoring based on sonarcloud suggestions

**Review**
@reddyashish 
@QilongTang

### CODE REVIEW

File: src/resources/LibraryStyles.css [hunk 1]
```diff
@@ -429,27 +429,12 @@ button {
...
 	background-color: transparent;
 	transition: 0.15s;
-}
-
-.SearchBar button {
-	color: #ddd;
-	transition: 0.15s;
-}
-
-.SearchBar button {
```
Comment (QilongTang) [inline src/resources/LibraryStyles.css:446]: this is not used?

Comment (Enzo707) [reply src/resources/LibraryStyles.css:446]: @QilongTang the `.SearchBar button` selector was repeated several times so I combined all styles applied. Those I've left there are the used ones

Comment (RobertGlobant20) [reply src/resources/LibraryStyles.css:446]: by checking the code I can see that the value
`color: #ddd;`
is set several times to different values, then if you combined all the styles for this button, how did you know which is the right color?

Comment (Enzo707) [reply src/resources/LibraryStyles.css:446]: @RobertGlobant20  If load the component using the browser, opening devtools you will see the styles which are actually in use.
Image 30-10-23 at 12 50

**Review by QilongTang — APPROVED** [review]

**Pull request merged**